documentation/API_documentation/kyc-tenure-API-Readiness-Checklist.md

```diff
@@ -1,12 +1,12 @@
 # API Readiness Checklist
 
-Checklist for kyc-tenure v0.1.0-rc.1 in r1.1.
+Checklist for kyc-tenure v0.1.0 in r1.2.
 
 | Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
 |  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | [relative link](/code/API_definitions/kyc-tenure.yaml) |
-|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y   | [r2.1](https://github.com/camaraproject/Commonalities/releases/tag/r2.1) |
-|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   | [r2.1](https://github.com/camaraproject/IdentityAndConsentManagement/releases/tag/r2.1) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y   | [r2.2](https://github.com/camaraproject/Commonalities/releases/tag/r2.2) |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   | [r2.2](https://github.com/camaraproject/IdentityAndConsentManagement/releases/tag/r2.2) |
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |   |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | Embedded documentation into API spec [relative link](/code/API_definitions/kyc-tenure.yaml) |
 |  6 | User stories                                 |   O   |         O         |    O    |    M   |  tbd |   |
```

Comment:
If not planned for this public release the status is "N" (it's optional for initial versions):
```suggestion
|  6 | User stories                                 |   O   |         O         |    O    |    M   |  N |   |
```